core/sonm.go

```diff
@@ -0,0 +1,164 @@
+package core
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/sonm-io/core/accounts"
+	"github.com/sonm-io/core/proto"
+	"github.com/sonm-io/core/util"
+	"github.com/sonm-io/core/util/xgrpc"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	dwhUpdateInterval = 5 * time.Second
+)
+
+var accountSlotsDefaults = map[uint64]uint64{
+	0: 16,
+	1: 32,
+	2: 64,
+	3: 128,
+	4: 256,
+}
+
+type SonmConfig struct {
+	KeyPath     string
+	KeyPass     string
```

Comment:
AFAIK, geth-node in mining mode already uses a keystore, can we use it instead of opening another one?

Reply:
these option useful without mining mode.
but there, geth have code for workin with key. right.